### PR TITLE
Create Greeting.java & print "Hello, World!"

### PR DESCRIPTION
**Pull request by andersonmc2**
Implemented "Hello, World!" print feature. Let me know if I need to add anything else.

### CODE REVIEW

**Comment by fenwickjb** [comment]
Would be better if testing were automated and not relying on humans who are error-prone. ;-)
Check this out...
http://stackoverflow.com/questions/1119385/junit-test-for-system-out-println

Also, it might help if you assign me as a "reviewer" (see panel to the right).

**Comment by andersonmc2** [comment]
I added a JUnit test for the output of Greeting and a makefile for convenience. Hopefully this is what you meant in your earlier comment.

File: GreetingTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();    
+
+    @Before
+    public void setUpStreams() {
+        System.setOut(new PrintStream(outContent));
+    }
+    
+    @After
+    public void cleanUpStreams() {
+        System.setOut(null);
```
Comment (fenwickjb) [inline GreetingTest.java]: Effectively "breaks" System.out for any subsequent use.

Each test should prepare whatever they need, then do their thing, then restore everything back to the way it was  prior to the test.

**Review by fenwickjb — CHANGES_REQUESTED** [review]

**Comment by andersonmc2** [comment]
I modified the JUnit test to restore System.out after testing.

**Pull request merged**